diff -Nur apvlv-0.1.2-Source.orig/src/ApvlvUtil.cpp apvlv-0.1.2-Source/src/ApvlvUtil.cpp
--- apvlv-0.1.2-Source.orig/src/ApvlvUtil.cpp	2011-06-24 15:30:12.000000000 +0800
+++ apvlv-0.1.2-Source/src/ApvlvUtil.cpp	2011-07-31 16:22:08.786230276 +0800
@@ -58,8 +58,8 @@
 string iconreg = string (DOCDIR) + "/icons/reg.png";
 string icondir = string (DOCDIR) + "/icons/dir.png";
 string iconpdf = string (DOCDIR) + "/icons/pdf.png";
-string inifile = "~/.apvlvrc";
-string sessionfile = "~/.apvlvinfo";
+string inifile = "~/.config/apvlv/apvlvrc";
+string sessionfile = "~/.config/apvlv/apvlvinfo";
 #endif
 
 // Converts the path given to a absolute path.
